packages-automation/auto-paymentAlert/package.json

```
@@ -10,6 +10,8 @@
     "auto-common": "*"
   },
   "scripts": {
-    "start" : "ts-node --files ./main.ts -T"
+    "start": "ts-node --files ./main.ts -T",
+    "dtsgen": "npx @kintone/dts-gen",
```

```suggestion
    "dtsgen": "npx @kintone/dts-gen --base-url https://rdmuhwtt6gx7.cybozu.com/ --app-id  249--type-name Record --namespace DB -o types/kintone.data.d.ts",
```

今からになっていますが、アプリ一つしかないので、型定義の取得はスクリプトではなく、直接した方がシンプルだと思います。

実行するとき、フォルダー内に `npm run dtsgen -- -u USER  -p PASSWORD`

---

承知しました。
ご指摘ありがとうございます。